tensorflow/compiler/tf2tensorrt/segment/segment.cc

```diff
@@ -781,15 +779,36 @@ Status SegmentGraph(const Graph* tf_graph,
     AddSegmentForNode(graph_properties, &node_segments, node, *device_name,
                       options.use_implicit_batch);
   }
-  string msg = StrCat(
-      "There are ", num_unsupported_ops, " ops of ", unsupported_ops.size(),
-      " different types in the graph that", " are not converted to TensorRT: ");
-  for (const auto& elem : unsupported_ops) {
-    StrAppend(&msg, elem, ", ");
+  string unsupported_op_report = StrCat(
+    "\n", string(80, '#'), "\n", "TensorRT unsupported/unconverted OP Report:");
+  int total_unconverted_ops{0};
+
+  // Copy key-value pair from unsupported_ops_map to vector of pairs
+    std::vector< std::pair<std::string, int> > _vect;
+  for (auto& _it : unsupported_ops_map) {
+      _vect.push_back(_it);
   }
-  LOG(INFO) << msg << "(For more information see "
-            << "https://docs.nvidia.com/deeplearning"
-            << "/frameworks/tf-trt-user-guide/index.html#supported-ops).";
+
+  // Sort in descending order, OP with the largest number of instances first
```

Comment:
Let's modify this to:
Sort in descending order using the number of uses of the OP that are not converted.

Reply:
Fixed